package.json

```
@@ -14,7 +14,7 @@
     "playground": "node ./scripts/playground.js",
     "prerelease": "pnpm build",
     "release": "changeset publish",
-    "test": "jest",
+    "test": "jest --projects packages/react-router --watch",
```

```suggestion
    "test": "jest",
```

Please revert this. You can do the same thing on the CLI with `npm test -- --projects packages/react-router --watch`